servers/quarkus-server/src/test/java/org/projectnessie/server/RestGitTest.java

```diff
@@ -192,4 +191,14 @@ public void testOptimisticLocking() {
     Assertions.assertNotEquals(b3.getHash(), newHash);
   }
 
+  static void assertEquals(Contents expected, Contents actual) {
```

Comment:
Mind renaming it to assertContentsEquals so it doesn't conflict with Assertions.assertEquals?

Reply:
fixed